boost-gradle/src/main/groovy/io/openliberty/boost/gradle/tasks/BoostPackageTask.groovy

```diff
@@ -195,13 +196,15 @@ public class BoostPackageTask extends AbstractBoostTask {
     protected void generateServerConfigEE() throws GradleException {
         String warName = null
 
-        if (project.war != null) {
+        if (project.plugins.hasPlugin('war')) {
 
             if (project.war.version == null) {
                 warName = project.war.baseName
             } else {
                 warName = project.war.baseName + "-" + project.war.version
             }
+        } else {
+            warName = getWarNameFromBoostApps()
```

Comment:
Should we be able to install multiple wars if we find them? Also install war both from the project and dependencies?

Reply:
Changed this to use a list of war names.